### PR TITLE
Portfolio project list

### PR DESCRIPTION
**Pull request by chiedozie013**
In this branch I:
- worked on the Works Section of the portfolio,
- also worked on the About Me section of the portfolio,
- also ensured my linters were all checked ok.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @chiedozie013 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with a readme that does not have the usage section. Kindly add it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by souad988 — COMMENTED** [review]
Hi @chiedozie013 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly add the missing section to your readme file as requested by the previous reviewer as only the FAQ section is optional from the microverse readme template . 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+              <a href="#"><i class="fab fa-linkedin"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works-section">
+        <h1>My Recent Works</h1>
+        <img src="./images/Img Placeholder.png" alt="workImg" />
+        <div class="workDetails">
+          <h2>Multi-Post Stories</h2>
```
Comment (brainconnect93) [inline index.html:70]: - [ ] I would kindly suggest you wrap the other section i.e `My Recent  Works` and `About me` inside of another `hx` tag apart from `h1`. Preferably `h2`. In a webpage it is advisable to have only one `h1` tag and this has been used in the header page. This is for semantic and SEO reasons. :+1: :+1:

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+
+body {
+  background-color: #fff;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 375px) {
```
Comment (brainconnect93) [inline styles.css:17]: - [ ] Great effort working on the design of this milestone. I would like you to work on the button in the About section and be sure it looks like the one we have in the Figma design.
- [ ] Also kindly give all The button (e.g., "See project" and "Get my resume") is a link a or button tag, and follow the button styling: `Active`, `Hover` and `Pressed` (see design)

Your Design :point_down: 

Screenshot_20230707-165819_Chrome

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @chiedozie013, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
-  Create the project cards and well aligned✔️
- Create the About me section with the tech stack ✔️
- Descriptive and professional README file ✔️
- Good use of GitHub flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by stanleySimeon — APPROVED** [review]
# Hi @chiedozie013, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:

#Congratulations! 🎉
 


## Highlights
- Good PR description
- Meaningful commit messages
- Linters check have passed

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**